functions-python/export_csv/function_config.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "export-csv",
+  "description": "Export the DB feed data as a csv file",
+  "entry_point": "export_and_upload_csv",
+  "timeout": 20,
+  "memory": "1Gi",
+  "trigger_http": true,
+  "include_folders": ["helpers", "dataset_service"],
+  "include_api_folders": ["utils", "database", "feed_filters", "common", "database_gen"],
+  "secret_environment_variables": [
+    {
+      "key": "FEEDS_DATABASE_URL"
+    }
+  ],
+  "ingress_settings": "ALLOW_INTERNAL_AND_GCLB",
+  "max_instance_request_concurrency": 20,
+  "max_instance_count": 10,
```

Let's use set the number of instances to one:
```suggestion
  "max_instance_request_concurrency": 1,
  "max_instance_count": 1,
```

---

Done